ui-kit.code-workspace

```diff
@@ -40,26 +40,10 @@
       "name": "📦 Atomic Hosted Page",
       "path": "./packages/atomic-hosted-page"
     },
-    {
-      "name": "📄 Atomic Storybook documentation",
-      "path": "./utils/atomic-storybook"
-    },
-    {
-      "name": "🚀 IIFE samples",
-      "path": "./packages/samples/iife"
-    },
-    {
-      "name": "🚀 Stencil samples",
-      "path": "./packages/samples/stencil"
-    },
     {
       "name": "🚀 React samples (Headless)",
       "path": "./packages/samples/headless-react"
     },
-    {
```

Comment:
Removed this obsolete entry - rest of changes are reordering to a more logical order.

Reply:
thanks for updating this! I've been using the workspace all the time.